.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Kubernetes-sigs/krew CI
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
+
+jobs:
+
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    
+    - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
+      run: |
+        set -euo pipefail
+        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+    
+    - name: Install dependencies
+      run: go mod download
+    
+    - name: Install gox
+      run: hack/install-gox.sh
+      
+    - name: Verify code quality
+      run: |
+        set -euo pipefail
+        go mod tidy 
+        git diff --no-patch --exit-code
+        hack/verify-code-patterns.sh
+        hack/verify-boilerplate.sh
+        hack/run-lint.sh
+        
+    - name: Unittest and coverage
+      run: go test -short -coverprofile=coverage.txt -covermode=atomic ./...
+        
+    - name: Make binaries && verify krew installation
+      run: hack/make-all.sh
+        
+    - name: Ensure kubectl installed
+      run: hack/ensure-kubectl-installed.sh
+        
+    - name: Integration tests
+      run: |
+        set -euo pipefail
+        hack/verify-installation.sh
+        hack/run-integration-tests.sh
+    
+    - name: Receipts upgrade migrations
```

Comment:
You don't need names for these, not only they aren't quite accurate, they're just repeating what's in the script name.

Reply:
You are right, but all steps need a name and it is part of keeping the entire structure of the pipeline a little clear, later you can add some other command, among other things.